### PR TITLE
Update kubes to 1.7.0

### PR DESCRIPTION
**Pull request by Fresheyeball**
###### Motivation for this change
Need access to breaking changes from the Kubernetes api, present in the new version.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/cluster/kubernetes/default.nix [hunk 1]
```diff
@@ -40,6 +40,11 @@ stdenv.mkDerivation rec {
     patchShebangs ./hack
   '';
 
+  preBuild = ''
+    echo "verion:"
```
Comment (Mic92) [inline pkgs/applications/networking/cluster/kubernetes/default.nix]: this should be probably removed again.

Comment (Fresheyeball) [reply pkgs/applications/networking/cluster/kubernetes/default.nix]: Done!

**Comment by LnL7** [comment]
FYI. The k8s service also needs changes in order for that to work with a newer version. See https://github.com/NixOS/nixpkgs/pull/25426

**Comment by Fresheyeball** [comment]
@LnL7, I don't see that package.

**Comment by globin** [comment]
ping @offlinehacker @matejc

**Comment by matejc** [comment]
@Fresheyeball have you got a plan to upgrade the kubernetes service? If yes, then I suggest adding WIP tag here on github and some plan of attack, otherwise this should not be merged.

**Comment by offlinehacker** [comment]
I'm closing this in favour of https://github.com/NixOS/nixpkgs/pull/25426

**Pull request closed without merging**